src/components/auth/RegisterForm.jsx

```
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { Box, Button, TextField, Typography, Link, Alert, CircularProgress } from '@mui/material';
```

ESLint

[error] 'Typography' is defined but never used.

(no-unused-vars)